### PR TITLE
Fix ,rounding issue with + and - number input

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: src/NumericInput.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ export default class NumericInput extends Component {
     };
 
     static defaultProps = {
+        decimal: 2,
```
Comment (borisyankov) [inline src/NumericInput.js]: +s ? 
decimals?

File: src/NumericInput.js [hunk 2]
```diff
@@ -26,18 +28,18 @@ export default class NumericInput extends Component {
...
         onChange({ target: { value: newValue } });
     }
 
     onStepDown = () => {
-        const { step, onChange } = this.props;
+        const { step, onChange, decimal } = this.props;
         const { value } = this.state;
-        const newValue = +value - step;
-        this.setState({ value: newValue});
+        const newValue = +(+value - step).toFixed(decimal);
```
Comment (borisyankov) [inline src/NumericInput.js]: The + sign outside does not seem needed?
The +value - step is sure to be number if step is number, and if itsn't there is a bigger problem :)

File: src/NumericInput.js [hunk 3]
```diff
@@ -26,18 +28,18 @@ export default class NumericInput extends Component {
     }
 
     onStepUp = () => {
-        const { step, onChange } = this.props;
+        const { step, decimal, onChange } = this.props;
         const { value } = this.state;
-        const newValue = +value + step;
-        this.setState({ value: newValue});
+        const newValue = +(+value + step).toFixed(decimal);
```
Comment (borisyankov) [inline src/NumericInput.js]: Same as lower comment.

File: src/NumericInput.js (hunk 2 above)
Comment (qingweibinary) [inline src/NumericInput.js]: toFixed convert number to string is not mistaken, I shall add parenthesis so the precedence is more explicit though

File: src/NumericInput.js (hunk 1 above)
Comment (qingweibinary) [inline src/NumericInput.js]: Sure

File: src/NumericInput.js (hunk 2 above)
Comment (borisyankov) [inline src/NumericInput.js]: Do we want to keep it in string though, or we lose digits? 123.00 in number is 123

Comment (qingweibinary) [inline src/NumericInput.js]: It was in number, and has been working, I lean toward to keep it in number

**Pull request merged**